BUILD

```diff
@@ -1,26 +1,21 @@
-load("@io_bazel_rules_go//go:def.bzl", "gazelle", "go_binary", "go_library", "go_prefix")
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_prefix")
+load("@bazel_gazelle//:def.bzl", "gazelle")
 
 gazelle(
     name = "gazelle",
-    build_tags = [
-        "container_image_ostree_stub",
-        "containers_image_openpgp",
-    ],
-    external = "vendored",
-    mode = "fix",
-    prefix = "github.com/GoogleCloudPlatform/container-diff",
+    prefix = "github.com/example/project",
```

Comment:
not totally sure but I don't think you want this here :) looks like it got propagated to the other BUILD files too

Reply:
haha, fixed.